torchvision/transforms.py

```diff
@@ -380,3 +380,41 @@ def __call__(self, img):
         scale = Scale(self.size, interpolation=self.interpolation)
         crop = CenterCrop(self.size)
         return crop(scale(img))
+
+
+class FiveCrop(object):
+    """Crop the given PIL.Image into four corners and the central crop."""
+
+    def __call__(self, img):
+        w, h = img.size
+        crop_w = w // 2
+        crop_h = h // 2
+
+        tl = img.crop((0, 0, crop_w, crop_h))
+        tr = img.crop((w - crop_w, 0, w, crop_h))
+        bl = img.crop((0, h - crop_h, crop_w, h))
+        br = img.crop((w - crop_w, h - crop_h, w, h))
+        center = CenterCrop((crop_h, crop_w))(img)
+        return [tl, tr, bl, br, center]
```

Comment:
Because there might be a misalignment on how the targets and the inputs will be handled by the dataloader by default, I think it might be worth a mention in the doc saying that this returns a tuple, and thus should be handled differently.
Also, could you return a tuple instead of a list?